src/api/app/models/user.rb

```diff
@@ -804,7 +804,7 @@ def to_param
     to_s
   end
 
-  # TODO: Remove once responsive_ux is out of beta
+  # TODO: Remove once notifications_redesign is out of beta
```

Comment:
nitpick: *if* we replace tasks. I don't think this comment makes much sense...

Reply:
Yeah... I wasn't sure. I'll remove it completely.

Reply:
As for the `Tasks` link in the top navigation, it's only shown if the `notifications_redesign` feature isn't enabled. If the tasks stay, then the changes in this PR will have to be adapted. Time will tell...